build.gradle

```diff
@@ -204,6 +204,9 @@ subprojects { subproject ->
             }
         }
     }
+    if(project.name != "app" && project.name != "gradle" && project.name != "dependencies") {
```

Comment:
What's this for? Does this belong in the PR?

Reply:
No it was added on accident I removed it now